srcalc/src/main/webapp/WEB-INF/views/enterVariables.jsp

```diff
@@ -21,14 +21,24 @@
     <c:set var="displayItemParam" scope="request" value="${displayItem}"/>
     <tr>
         <c:url var="qMarkImageUrl" value="/css/images/qmark.png"/>
-        <td class="attributeName"><c:out value="${displayItem.displayName}"/>:
+        <c:url var="noteImageUrl" value="/css/images/note.png"/>
+        <td class="attributeName">
+            <c:out value="${displayItem.displayName}"/>:
             <c:if test="${displayItem.helpTextAsHtml != ''}">
                 <a class="helpTextToggler"><img src="${qMarkImageUrl}" alt="?"/></a>
             </c:if>
+            <c:if test="${not empty displayItem.referenceInfo}">
+                <a class="referenceInfoToggler"><img src="${noteImageUrl}" alt="?"/></a>
+            </c:if>
         </td>
         <td class="attributeValue">
             <jsp:include page="fragments/${displayItem.fragmentName}"/>
 			<div class="variableDef">${displayItem.helpTextAsHtml}</div>
+            <div class="referenceNotes">
+                <!-- Unfortunately this line needs to be long because textarea tags interpret
+                white space literally. -->
+                <textarea disabled style="width: 700px; height:200px; text-indent:0px;"><c:forEach var="note" items="${displayItem.referenceInfo}">${note}</c:forEach></textarea>
```

Comment:
The disabled textarea looks weird in IE11 and doesn't even let you scroll in IE8.